cmd/pod-scaler/testdata/zz_fixture_TestMutatePodResources_resources_to_add.yaml

```diff
@@ -0,0 +1,65 @@
+  &v1.Pod{
```

Comment:
This doesn't seem to be yaml, where does it come from, come cmp output?

Reply:
Yeah, it's a `cmp.Diff()` output - I was wondering the best way to do this ... basically the point of the `mutate` func is to apply some diff from a -> b ... it seemed like a good way to test that in a concise way would be to assert something about that diff. Asserting that the output object is in some specific shape runs into the issue you pointed out in the last PR - without the source object, it's hard to reason about.

Reply:
Just serialize it at the end?

Reply:
If I serialize it, all the assert is about is the state of the Pod after we mutate the requests. That's not really useful, as you pointed out on the previous PR, as the important thing is the _change_ we make to the requests

Reply:
It does seem a bit weird that it would exist as a yaml file since it isn't yaml. You couldn't just make the "expected" as a `v1.Pod` struct instance and diff that with the actual result?

Reply:
The YAML bit is the diff library trying to be helpful - I can add a patch to make this a `.diff` if that would be useful.